modules/rules_elm/1.0.0/presubmit.yml

```diff
@@ -0,0 +1,24 @@
+# We recommend included a bcr test workspace that exercises your ruleset with bzlmod.
+# For an example, see https://github.com/aspect-build/bazel-lib/tree/main/e2e/bzlmod.
+bcr_test_module:
+  module_path: "examples/elm-test"
+  matrix:
+    platform: ["ubuntu2004"]
+    bazel: [7.x]
+  tasks:
+    run_tests:
+      name: "Run test module"
+      platform: ${{ platform }}
+      bazel: ${{ bazel }}
+      environment:
+        # The Nix installer updates ~/.profile to set PATH. However, the Bazel
+        # CI setup seems to disregard ~/.profile. So we set PATH manually here.
+        # PATH: /var/lib/buildkite-agent/.nix-profile/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+      shell_commands:
+        # - curl -L https://nixos.org/nix/install | sh -s -- --no-daemon
+        # try without nix
+        - rm examples/elm-test/.bazelrc
+        - echo "build --experimental_enable_bzlmod" >> examples/elm-test/.bazelrc
+        - echo "build --registry=file:///workdir" >> examples/elm-test/.bazelrc
```

Comment:
I know this is ugly but I'd like to check if build will pass without nix. When it would, I'll replace it with a patch.

Reply:
This change got reverted. Build is green now: https://github.com/bazelbuild/bazel-central-registry/pull/2833#issuecomment-2370868624